content/blog/2023/10/camel41-whatsnew/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Apache Camel 4.1 What's New"
+date: 2023-10-11
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 4.1 release.
+---
+
+Apache Camel 4.1 (non LTS) has just been [released](/blog/2023/10/RELEASE-4.1.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+## DSL
+
+The XML and YAML DSL now has better support for defining <bean> which can be configured with properties, and
```

Comment:
```suggestion
The XML and YAML DSL now has better support for defining `<bean>` which can be configured with properties, and
```

Reply:
This causes build failure when using bean in quotes